### PR TITLE
Add case that check qga.service status after remove it.

### PR DESCRIPTION
**Pull request by 6-dehan**
ID: 1887388
Signed-off-by: Dehan Meng <demeng@redhat.com>

### CODE REVIEW

**Comment by 6-dehan** [comment]
@vivianQizhu please review when you're free. thanks

File: qemu/tests/qemu_guest_agent.py [hunk 1]
```diff
@@ -2438,7 +2438,28 @@ def check_value_frontend_open(out, expected):
...
+            if self._check_ga_service(session, params.get("gagent_status_cmd")):
+                test.fail("QGA service should be removed.")
+        finally:
+            error_context.context("Recover test env that start qga.", logging.info)
+            self.gagent_install(session, self.vm)
+            self.gagent_start(session, self.vm)
+            self.gagent_verify(params, self.vm)
+
+    @error_context.context_aware
+    def gagent_che_frozen_io(self, test, params, env):
```
Comment (vivianQizhu) [inline qemu/tests/qemu_guest_agent.py:2462]: This line modification should be a mistake.

File: qemu/tests/cfg/qemu_guest_agent.cfg [hunk 2]
```diff
@@ -4,6 +4,7 @@
     type = qemu_guest_agent
     gagent_name = "org.qemu.guest_agent.0"
     gagent_install_cmd = "yum install -y qemu-guest-agent"
+    gagent_uninstall_cmd = "yum -y remove qemu-guest-agent"
```
Comment (vivianQizhu) [inline qemu/tests/cfg/qemu_guest_agent.cfg:7]: `yum remove` will remove the dependency, which might be used by other packages, `rpm -e --nodeps` might be safer.

**Comment by 6-dehan** [comment]
oh, nice idea, thanks.

**Review by vivianQizhu — APPROVED** [review]
Test result please.

**Comment by 6-dehan** [comment]
> Test result please.

hi qianqian, 
this auto bug is related product bug bz_1882719 that its status still be post,so the test result should be failed with current rpm package so far,so I fix this auto bug with expected result . of course if you have doubt or you wanna confirm with a accurate result, we can wait for product bz status turns to verified. but this result we can be clear ,btw,so what do you think?

**Comment by vivianQizhu** [comment]
@6-dehan No we don't have to, let have this in first. It could be twist if needed after the product bz fixed.

**Comment by 6-dehan** [comment]
> @6-dehan No we don't have to, let have this in first. It could be twist if needed after the product bz fixed.

okay,nice thanks qianqian.

**Comment by 6-dehan** [comment]
@vivianQizhu rhel8.3.1 qga pkg has released ,and following are results.

(1/3) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1446.73 s)
 (2/3) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.rh_kernel_update.q35: PASS (112.35 s)
 (3/3) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.1.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_qgastatus_after_remove_qga.q35: PASS (91.38 s)

**Comment by vivianQizhu** [comment]
@6-dehan Well noted, thanks.

**Comment by 6-dehan** [comment]
> @6-dehan Well noted, thanks.

My pleasure, thanks for your help to review.

**Pull request merged**